source/_components/switch.template.markdown

```diff
@@ -35,7 +36,12 @@ switch:
           service: switch.turn_off
           data:
             entity_id: switch.skylight_close
+        availability_template: >-
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.